distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,363 @@
+
+${ddf-registry} contains the base registry components, plugins, sources and interfaces needed for ${branding} to function as a registry connecting multiple nodes.
+
+=== ${ddf-registry} Prerequisites
+
+To use the ${ddf-registry}, the following apps/features must be installed:
+
+* <<_managing_ddf_spatial,${ddf-spatial}>>
+
+=== Installing ${ddf-registry}
+
+${ddf-registry} is not installed by default with a standard installation.
+
+To install:
+
+* Navigate to the ${admin-console}.
+* Click the *Manage* applications button.
+* Click the *Install* icon.
+* ${ddf-registry} will move to *Active Applications* upon startup.
+
+=== Configuring ${ddf-registry}
+
+These configurations are available from the ${ddf-registry} application in the ${admin-console}.
```

Comment:
Some of the configurations looked a bit different that what is current, but it may have been an older PR of the docs...

RegistryStore
FederationAdminMBean
FederationAdminService
RegistryPublicationService